services/wordpress/wordpress-downloads.service.js

```diff
@@ -74,21 +79,25 @@ function InstallsForExtensionType(extensionType) {
       return `Wordpress${capt}Installs`
     }
 
-    static get extensionType() {
-      return extensionType
-    }
-
     static get category() {
       return 'downloads'
     }
 
-    static render({ response }) {
+    static render({ installCount }) {
       return {
-        message: `${metric(response.active_installs)}+`,
```

Comment:
I may be getting tricked by GH's web dif again, but is this changing the message to drop the trailing `+` suffix?

Reply:
Yes, that change I’ll own up to. It’s the only badge that shows this, and it’s something that’s true of all download and install badges (for various reasons), so I don’t think it makes sense to include it.

Reply:
Agreed. It was a little confusing to have that in here before, at least relative to all the other download badges